fms/utils/tokenizers.py

```diff
@@ -67,6 +70,10 @@ def convert_ids_to_tokens(self, ids: torch.LongTensor):
         return [chr(i) for i in ids]
 
     def convert_tokens_to_ids(self, tokens: Union[str, list[str]]):
+        if isinstance(tokens, str) and len(tokens) == 1:
```

Comment:
we might want to explicitly raise a descriptive error in the `len(tokens)!=1` case, otherwise below we'll get an error like `ord() expected a character, but string of length 2 found` which might not be clear to the user